### ISSUE TITLE
`stats` not working in mac

### ISSUE THREAD

**Issue by pr2tik1** [pr2tik1/termipy#10]
## Command
```
@termipy >> stats
```

## Output / Error Log
```
Error processing command: '<' not supported between instances of 'NoneType' and 'float'
```

**Comment by pr2tik1** [comment]
Works fine on Linux based OS.

```
@termipy >> stats
```

```
┌───────────────────────────────────────────┐
│ CPU Usage                                 │
├───────────────────────────────────────────┤
│ CPU Usage: 27.0%                          │
└───────────────────────────────────────────┘

┌───────────────────────────────────────────┐
│ Memory Usage                              │
├───────────────────────────────────────────┤
│ Memory Usage: 49.5%                       │
│ Total Memory: 62.79 GB                    │
│ Available Memory: 31.69 GB                │
└───────────────────────────────────────────┘

┌───────────────────────────────────────────┐
│ Disk Usage                                │
├───────────────────────────────────────────┤
│ Disk Usage: 72.1%                         │
│ Total Disk Space: 1399.98 GB              │
│ Free Disk Space: 390.91 GB                │
└───────────────────────────────────────────┘

┌───────────────────────────────────────────┐
│ Network Usage                             │
├───────────────────────────────────────────┤
│ Network Bytes Sent: 2.17 MB               │
│ Network Bytes Received: 1.45 MB           │
└───────────────────────────────────────────┘

┌───────────────────────────────────────────┐
│ Process Usage (Top-5)                     │
├───────────────────────────────────────────┤
│ PID: 313, Name: node, CPU: 10.0%          │
│ PID: 69, Name: node, CPU: 1.5%            │
│ PID: 35, Name: supervisor, CPU: 0.5%      │
│ PID: 348, Name: node, CPU: 0.5%           │
│ PID: 1, Name: supervisor, CPU: 0.0%       │
└───────────────────────────────────────────┘

Press Ctrl+C to STOP monitoring.
```

**Comment by pr2tik1** [comment]
┌───────────────────────────────────────────┐
│ Process Usage                             │
├───────────────────────────────────────────┤
│ Error: '<' not supported between instances of 'NoneType' and 'float' │
└───────────────────────────────────────────┘
